splight_lib/models/datalake.py

```diff
@@ -0,0 +1,155 @@
+from datetime import datetime
+from enum import Enum
+from hashlib import sha256
+from typing import Annotated, Any, Generic, Literal, Self, TypeVar
+
+from pydantic import BaseModel, ConfigDict, Field, PrivateAttr
+
+from splight_lib.client.datalake import DatalakeClientBuilder
+from splight_lib.client.datalake.abstract import AbstractDatalakeClient
+from splight_lib.client.datalake.constants import StepName
+from splight_lib.models.asset import Asset
+from splight_lib.models.attribute import Attribute
+from splight_lib.models.exceptions import TraceAlreadyExistsError
+from splight_lib.settings import settings
+
+T = TypeVar("T")
+
+
+def hash(string: str) -> str:
+    return sha256(string.encode("utf-8")).hexdigest()
+
+
+def get_datalake_client() -> AbstractDatalakeClient:
+    return DatalakeClientBuilder.build(
+        dl_client_type=settings.DL_CLIENT_TYPE,
+        parameters={
+            "base_url": settings.SPLIGHT_PLATFORM_API_HOST,
+            "access_id": settings.SPLIGHT_ACCESS_ID,
+            "secret_key": settings.SPLIGHT_SECRET_KEY,
+            "buffer_size": settings.DL_BUFFER_SIZE,
+            "buffer_timeout": settings.DL_BUFFER_TIMEOUT,
+        },
+    )
+
+
+class TraceType(str, Enum):
+    QUERY = "QUERY"
+    EXPRESSION = "EXPRESSION"
+    METADATA = "METADATA"
+
+    @classmethod
+    def choices(cls):
+        return tuple((i.name, i.value) for i in cls)
```

Comment:
I think this is already defined on `splight_lib/client/datalake/constants.py`

Reply:
Ups, forgot to remove it from constants.